Octokit/Models/Request/RepositoryTransfer.cs

```diff
@@ -0,0 +1,70 @@
+using System;
+using System.Text;
+using System.Diagnostics;
+using System.Globalization;
+using System.Collections.Generic;
+
+namespace Octokit
+{
+    /// <summary>
+    /// Describes the transfer of a repository to a new owner.
+    /// </summary>
+    [DebuggerDisplay("{DebuggerDisplay,nq}")]
+    public class RepositoryTransfer
+    {
+        /// <summary>
+        /// Creates a new repository transfer description with no team Ids.
+        /// </summary>
+        /// <param name="newOwner">The new owner of the repository after the transfer.</param>
+        /// <remarks>
+        /// This is the only valid constructor for repositories whose current owner is not an
+        /// organization.
+        /// </remarks>
+        public RepositoryTransfer(string newOwner)
+        {
+            Ensure.ArgumentNotNullOrEmptyString(newOwner, nameof(newOwner));
+
+            NewOwner = newOwner;
+        }
+
+        /// <summary>
+        /// Creates a new repository transfer description.
+        /// </summary>
+        /// <param name="newOwner">The new owner of the repository after the transfer.</param>
+        /// <param name="teamId">A list of team Ids to add to the repository after the transfer.</param>
+        /// <remarks>
```

Comment:
The wording on this remark is a bit odd (what do you mean by "invalid description"?).  Also <remarks> objects aren't visibile in intellisense AFAIK, so I would suggest removing this remark, and instead what I normally do is type a brief explanation in brackets of the description of the param element itself 

eg
`/// <param name="teamId">A list of team Ids to add to the repository after the transfer (only applies to Organization owned repositories).</param>`